docs/src/howto-guides/027_remote_access_with_cumulocity.md

```diff
@@ -0,0 +1,41 @@
+# Cumulocity RemoteAcessConnect plugin
+
+To access a device remotely that runs thin-edge.io, a plugin of the operation plugin concept is used. The tedge-mapper is checking for cloud remote access operation and is triggering the particular plugin. You can use the remote access tab in device management to access the device via SSH or VNC.
+
+## Requirements
+
+- Working thin-edge.io installation
+
+- The Cloud Remote Access Feature is assigned to your Tenant. If not ask your Administrator to get it assigned to your Tenant. Please note that the Version must be at least 1007.2.0+
+
+- The Cloud Remote Access Role must be assigned to the user who wants to use that Feature: <em>Administration &rarr; Role &rarr; &lt;any Role&gt; &rarr; check "Remote Access"</em>. Assign the role to the user used for the next steps.
+
+
```

Comment:
The VNC or SSH server installation (based on what the user wants) is also a requirement, right?